packages/main-menu-react/src/contexts/mainMenuContext.tsx

```diff
@@ -1,19 +1,27 @@
 import { createContext, useContext } from "react";
 
 export type NavigationFunction = (path: string) => void;
-export type IsActiveFunction = (path: string) => boolean;
+export type IsActiveFunction = (path: string, currentOpenMenu: string) => boolean;
 interface MainMenuContextProps {
     navigationFunction: NavigationFunction;
     isActiveFunction: IsActiveFunction;
+    reduceMotion: boolean;
 }
 
 export const defaultNavigationFunction = (path: string) => {
     window && window.location.assign(path);
 };
+export const defaultIsActiveFunction = (path: string, currentOpenMenu: string) => {
+    const thisMenuIsOpen = path !== "" && currentOpenMenu.includes(path);
+    const itemPathMatches = window.location ? window.location.pathname.includes(path) : false;
```

Comment:
Trenger ikke denne en bedre windowsjekk for SSR?

Reply:
Hmm, det kan kanskje hende.. Så ut til å fungere fint når jeg bygget og servet portalen lokalt (ikke dev mode), men kan sikkert gjøres enda sikrere. Ellers tenker jeg at det er en typisk situasjon hvor man ville lagt inn sin egen funksjon.